dev/com.ibm.ws.cdi.weld/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -18,3 +18,12 @@ cdi12.desc=Defines the behavior of the Contexts and Dependency Injection (CDI) v
 
 cdi12.enableImplicitBeanArchives.name=Enable scanning implicit bean archives
 cdi12.enableImplicitBeanArchives.desc=The implicit bean archives are scanned for any bean discoveries.
+
+cdi=Contexts And Dependency Injection
+cdi.desc=Defines the behavior of the Contexts and Dependency Injection.
+
+cdi.enableImplicitBeanArchives.name=Enable scanning implicit bean archives. Applies to CDI 1.2 or newer.
+cdi.enableImplicitBeanArchives.desc=The implicit bean archives are scanned for any bean discoveries.
+
+cdi.emptyBeansXMLExplicitArchive.name=An Archive with an empty beans.xml file is considered to be an explicit bean archive. Applies to CDI 4.0 or newer.
+cdi.emptyBeansXMLExplicitArchive.desc=An archive with an empty beans.xml is considered an explicit bean archive. This is to maintain the existing behaviour prior to CDI 4.0.
```

Comment:
suggested edit

An archive with an empty beans.xml is an explicit bean archive. This property maintains the existing behavior before CDI 4.0.

Reply:
Ended up rewriting this description...
> CDI 4.0 and later considers an archive with an empty beans.xml file to be an implicit bean archive. This property maintains the existing behavior before to CDI 4.0.